src/blas.c

```diff
@@ -342,32 +342,32 @@ void fill_cpu(int N, float ALPHA, float *X, int INCX)
     }
 }
 
-void deinter_cpu(int NX, float *X, int NY, float *Y, int B, float *OUT)
+void deinter_cpu(int NX, float *X, int NY, float *Y, int B, float *OUTPUT)
```

Comment:
`OUT` was conflicting with a macro defined in a Windows header, renamed to `OUTPUT` to avoid the conflict.